maintainer/conda/environment.yml

```diff
@@ -27,6 +27,7 @@ dependencies:
   - tqdm>=4.43.0
   - sphinxcontrib-bibtex
   - sphinx_rtd_theme
+  - pyyaml>=3.02
```

Comment:
@jaclark5  what is this addition for exactly? I might have missed it.

Reply:
The new citation method won't compile without this constraint.